### PR TITLE
Add navbar

### PR DESCRIPTION
**Pull request by surfer19**

### CODE REVIEW

File: public/js/app.js [hunk 1]
```diff
@@ -93,32 +93,60 @@
...
+  activateNavBg(window.scrollY, header_height);
+};
+
+function activateNavBg(scrollpos, header_height) {
+  if (scrollpos >= header_height) {
+    add_class_on_scroll();
+  } else {
+    remove_class_on_scroll();
+  }
+} // Closes the sidebar menu
```
Comment (mjavorka) [inline public/js/app.js:123]: Do we still need this commented code?

File: resources/js/custom.js [hunk 2]
```diff
@@ -1,31 +1,57 @@
...
+    else {
+        remove_class_on_scroll()
+    }
+}
 // Closes the sidebar menu
-$("#menu-close").click(function(e) {
-    e.preventDefault();
-    $("#sidebar-wrapper").toggleClass("active");
-  });
+// $("#menu-close").click(function(e) {
```
Comment (mjavorka) [inline resources/js/custom.js:28]: And also here and below

File: public/js/app.js (hunk 1 above)
Comment (surfer19) [reply public/js/app.js]: done

File: resources/js/custom.js (hunk 2 above)
Comment (surfer19) [reply resources/js/custom.js]: done

**Pull request merged**